python/dgl/graphbolt/minibatch.py

```diff
@@ -208,15 +223,21 @@ def cast_to_minimum_dtype(v: CSCFormatBase):
                             ),
                         ),
                     )
-                    for etype, v in subgraph.sampled_csc.items()
-                }
                 num_src_nodes = {
-                    ntype: nodes.size(0)
-                    for ntype, nodes in original_row_node_ids.items()
+                    ntype: (
+                        original_row_node_ids[ntype].size(0)
+                        if original_row_node_ids.get(ntype) is not None
+                        else 0
+                    )
+                    for ntype in set(node_types)
```

Comment:
seems `node_types` should be a set instead list, why not directly instantiated as set at the beginning?